src/packageManagers/npm.ts

```diff
@@ -35,19 +35,21 @@ export default class NpmPackageManager implements PackageManager {
 
         // Add the authentication token for the registry hostname
         const registryUrl = new URL(url);
-        await asyncExec(`npm config set //${registryUrl.hostname}/:_authToken=${authToken}`);
+        const path = registryUrl.pathname.split("/").slice(0, -1).join("/");
+        await asyncExec(`npm config set //${registryUrl.hostname}${path}/:_authToken=${authToken}`);
```

Comment:
Don't you need a slash between the hostname and the path?

Like: ${hostname}/${path}

this comment applies to all package managers.

Reply:
No. It already has one it in the path.

Reply:
ok